### PR TITLE
[SYCL][Graph][DOC] add an edge case for record&replay mode

### PR DESCRIPTION
**Pull request by guoyejun**

### CODE REVIEW

**Comment by guoyejun** [comment]
any comment? thanks.

File: sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc [hunk 1]
```diff
@@ -273,15 +273,16 @@ been submitted.
...
 represent data dependencies between two command groups captured as nodes.
 Secondly, by using the `handler::depends_on()` mechanism inside a command group
 captured as a node. However, for an event passed to `handler::depends_on()` to
 create an edge, it must be an event returned from a queue
 submission captured by the same graph. Otherwise, a synchronous error will be
 thrown with error code `invalid`. `handler::depends_on()` can be
 used to express edges when a user is working with USM memory rather than SYCL
-buffers.
+buffers. Thirdly, for graph recorded with in-order queue, an edge is added
+automatically between two sequential command groups.
```
Comment (EwanC) [inline sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc:285]: Would tweak this slightly.
```suggestion
buffers. Thirdly, for a graph recorded with an in-order queue, an edge is added
automatically between two sequential command groups submitted to the in-order queue.
```

**Review by EwanC — APPROVED** [review]
LGTM, minor rewording suggestion.

**Comment by EwanC** [comment]
@guoyejun Could your rebase or merge `main` into this branch, I think that will work around the CI fail and we can get this merged.

**Comment by guoyejun** [comment]
@EwanC  the CI is passed, we may get this merged, thanks.

**Comment by EwanC** [comment]
@intel/llvm-gatekeepers Please merge, thanks

**Pull request merged**